src/Amazon.AspNetCore.DataProtection.SSM/Amazon.AspNetCore.DataProtection.SSM.csproj

```diff
@@ -42,25 +42,28 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.2" />
-    <PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="3.7.17.46" />
+    <PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.7" />
+    <PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="3.7.300.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.4">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection.Extensions" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection.Extensions" Version="3.1.32" />
```

Comment:
From my testing if you leave this at the base version for the TFM like 3.1.0 for `netcoreapp3.1` or 6.0.0 for .net6.0 then when you execute .NET publish it does not include this package in the project output and lets the application use the version that is available in the installed runtime. If you specify a specific version with in the family then it things you want to override the version installed with the runtime.

It is best if just say use the one that is installed with the runtime. So we should keep the versions at the base. Or better yet set the versions to something like `6.0.*`

Reply:
Thanks for the review. I have made the required changes to add wildcard `*` based package reference.

Also update the following package references for `Amazon.AspNetCore.DataProtection.SSM.Tests.csproj` (this targets `net6.0`):
- `AWSSDK.Extensions.NETCore.Setup` version `3.7.7`
- `AWSSDK.SimpleSystemsManagement` version `3.7.300.1`
- `Microsoft.AspNetCore.DataProtection.Extensions` version `6.0.*`